pom.xml

```diff
@@ -46,6 +46,16 @@
 			<artifactId>spring-boot-starter-test</artifactId>
 			<scope>test</scope>
 		</dependency>
+		<dependency>
```

Comment:
Please describe what functionalities from added dependencies are you using to secure the spring boot application.